core/Extensions/AppExtensions.cs

```diff
@@ -49,7 +49,8 @@ public static ContainerBuilder AddSystemCore(this ContainerBuilder builder, ICon
             var remoteNodes = configuration.GetSection("Node:Network:SeedList").GetChildren().ToArray();
             var node = new Node
             {
-                EndPoint = new IPEndPoint(Util.GetIpAddress(), 0),
```

Comment:
The old implementation used DNS Resolution everytime the local node object was created.